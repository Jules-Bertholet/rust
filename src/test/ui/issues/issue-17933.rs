pub static X: usize = 1;

fn main() {
    match 1 {
        self::X => { },
        //~^ ERROR referencing statics in patterns is experimental
        _       => { },
    }
}
