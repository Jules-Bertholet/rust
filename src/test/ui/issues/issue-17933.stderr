error[E0658]: referencing statics in patterns is experimental
  --> $DIR/issue-17933.rs:5:9
   |
LL |         self::X => { },
   |         ^^^^^^^
   |
   = help: add `#![feature(static_in_pattern)]` to the crate attributes to enable

error: aborting due to previous error

For more information about this error, try `rustc --explain E0658`.
